src/content/en/fundamentals/engage-and-retain/push-notifications/index.markdown

```diff
@@ -1,24 +1,77 @@
 ---
 layout: shared/narrow-pages-list
-title: "Push Notifications"
-description: "You may already know how to implement push notifications. Are you using them well? Could you be using them better?"
-published_on: 2016-01-05
-updated_on: 2016-01-05
+title: "Push Notifications: Timely, Relevant, and Precise"
+description: "Push notifications are one of the most valuable capabilities of native apps, and this capability is now available on the web. To get the most out of them, notifications need to be timely, precise, and relevant."
+published_on: 2016-07-01
+updated_on: 2016-07-01
 order: 1
-translation_priority: 0
+translation_priority: 1
 authors:
   - josephmedley
 ---
 
-<p class="intro">
-  You may already know <a href="/web/fundamentals/primers/push-notifications">how to implement push notifications</a>. Are you using them well? Could you be using them better? 
+<p class="intro">Don't annoy your users, or you'll lose access to them
+	forever. Do we need to say anything more than that? We do because it's easier said than
+done.
+
+Push notifications are one of the most valuable capabilities of native apps, and
+this capability is now available on the web. To get the most out of them,
+notifications need to be timely, precise, and relevant.
 </p>
 
-In 2015, [a case study](http://googlewebmastercentral.blogspot.com/2015/07/google-case-study-on-app-download-interstitials.html) 
-found that when prompted to download and install Google+, 9% of users did so. 
-That's an impressive click-through rate, or it would be if we were talking
-about online ads. The same study found that 69% of users both left the page 
-that prompted and ignored the invite to install Google+.
+<div class="mdl-grid">
+  <div class="mdl-cell mdl-cell--6-col" markdown="1">
+_Timely_—A timely notification is one that appears when users want it and when
+it matters to them.
+
+_Precise_—A precise notification is one that has specific information that can
+be acted on immediately.
+
+_Relevant_—A relevant message is one about people or subjects the user cares
+about.
+  </div>
+  
+  <figure class="mdl-cell mdl-cell--6-col">
+    <img src="images/tpnr.png" alt="Timely, precise, and relevant">
```

Comment:
Maybe the screenshot goes here, and the timely/precise/relevant circles go on the next page? Optional.